assets/src/editor/events/initialization/data-hydrator.js

```diff
@@ -136,7 +137,14 @@ const hydrateEntityData = async ( rawData, schemas ) => {
 					return { [ model ]: hydratedEntities };
 				}
 			);
-			resolveGetEntities( model );
+			if ( model === 'price_type' ) {
+				receiveEntityResponse(
+					model,
+					undefined,
+					Object.values( entities[ model ] ),
+				);
+				resolveGetEntities( model );
```

Comment:
this was previously being done for all entities but now it is only happening for prices?!?!?!
was this supposed to be after the closing brace that follows it? 
ie: we just need to receive price types prior to calling `resolveGetEntities( model )` ?

Reply:
We just need that for `price_type`, doing for others doesn't really matter, that's why I moved it inside that block.